server/hotrod/src/main/java/org/infinispan/server/hotrod/Encoder2x.java

```diff
@@ -724,36 +725,25 @@ private Optional<AbstractTopologyResponse> generateTopologyResponse(short client
          return Optional.empty();
       }
 
-      int topologyId = currentTopologyId;
-
       if (log.isTraceEnabled()) {
          log.tracef("Check for partial topologies: members=%s, endpoints=%s, client-topology=%s, server-topology=%s",
-               cacheMembers, cacheMembers, responseTopologyId, topologyId);
+               cacheMembers, cacheMembers, responseTopologyId, cacheTopology.getTopologyId());
```

Comment:
the last argument should be `cacheTopology.getReadConsistentHash().hashCode();` right?

Reply:
No, I was leaving it as the server topology. TBH I had debated about logging all 3, maybe I should do that instead.

Reply:
+1 to log all 3.
I also just noticed the `cacheMembers` are logged twice. The second should be `serverEndpoints` (?)

Reply:
Ah, yeah looks like it, haha.

Reply:
TBH cacheMembers isn't even needed to be logged as serverEndpoints has them as the key.

Reply:
Updated.